stack.yaml

```diff
@@ -0,0 +1,23 @@
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/master/snapshots/cardano-1.25.1.yaml
```

Comment:
If the stack build's not tested and you're only using cabal, best to remove `stack.yaml` and anything related to it.